packages/demo/src/app/post-sample/components/stepper/stepper-demo/stepper-demo.component.html

```diff
@@ -11,15 +11,12 @@ <h2 class="visually-hidden">Order progress, step 3 of 4</h2>
     <li
       *ngFor="let step of steps; index as i"
       [attr.aria-current]="isCurrent(step) ? 'step' : undefined"
-      class="stepper-item"
+      class="stepper-item stepper-interactive"
     >
-      <a *ngIf="i < currentIndex" class="stepper-link" (click)="currentIndex = i" href="#">
-        <span class="visually-hidden">Complete:</span>
+      <a class="stepper-link" (click)="currentIndex = i" href="#">
+        <span class="visually-hidden" *ngIf="i < currentIndex">Complete:</span>
```

Comment:
From an accessibility perspective, it makes no sense that the active step is a link, because clicking it would not have any effect. Should we use a `<span>` for the active item?